.github/workflows/build-test-e2e.yml

```diff
@@ -4,7 +4,7 @@ on: push
 jobs:
   build-and-run-e2e:
     if: ${{ github.ref_name == 'develop' || github.ref_name == 'release-**' || github.actor == github.repository_owner }}
-    runs-on: macos-latest
+    runs-on: ${{ github.repository_owner == 'Sage-Bionetworks' && 'ubuntu-22.04-4core-16GBRAM-150GBSSD' || 'macos-latest' }}
```

Comment:
Use larger runner when e2e tests are run in Sage-Bionetworks organization, but continue using `macos` runner in dev forks

Reply:
Good idea. Let's do this in the monorepo as well

Reply:
https://sagebionetworks.jira.com/browse/PORTALS-2798